packages/core/utils/shouldAuthorize.js

```diff
@@ -0,0 +1,23 @@
+import get from 'lodash/fp/get';
+import isNode from './isNode';
+
+/**
+ * Retrieve the bearer token cookie.
+ *
+ * @param {object} cookie Object containing valid cookies for the app.
+ */
+export const getBearerToken = (cookie) => (
```

Comment:
This seems like something that could either be included directly in the function below (e.g., `const authorizationBearerToken = get('authorizationBearerToken', cookie)` or split out into its own file as a default export.